autoware_planning_msgs/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/PathPoint.msg"
   "msg/RouteState.msg"
   "srv/ClearRoute.srv"
+  "srv/SetLaneChangeOverride.srv"
```

Comment:
Please move the message to the following repository.
https://github.com/tier4/tier4_autoware_msgs/tree/tier4/universe/tier4_planning_msgs/msg